### PR TITLE
add @media keyword to stylesheet exports

### PR DESCRIPTION
**Pull request by mktcode**
fixes part of #273

### CODE REVIEW

File: core-bundle/src/Resources/contao/classes/StyleSheets.php [hunk 1]
```diff
@@ -1391,7 +1391,7 @@ public function exportStyleSheet(DataContainer $dc)
 		// Add the media query (see #7560)
 		if ($objStyleSheet->mediaQuery != '')
 		{
-			$objFile->append($objStyleSheet->mediaQuery . ' {');
+			$objFile->append('@media ' . str_replace('@media ', '', $objStyleSheet->mediaQuery) . ' {');
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/classes/StyleSheets.php]: Adding `@media` in the "Media query" field will break the front end, therefore we do not need the `str_replace()`.

**Review by leofeyer — CHANGES_REQUESTED** [review]

**Comment by mktcode** [comment]
I don't understand. This method is only used to send a file to the browser, for the export functionality in the backend. Do I miss something here?

**Comment by leofeyer** [comment]
`$objStyleSheet->mediaQuery` can never contain `@media`, otherwise the generated `<style>` tag in the front end would be invalid. Therefore the `str_replace()` is unnecessary.

**Comment by mktcode** [comment]
Ahhh okay.... gotya!

**Comment by leofeyer** [comment]
Thank you @mktcode.

**Pull request merged**